src/features/keystone/SelectKeystoneAccountsScreen.tsx

```diff
@@ -73,26 +73,15 @@ const SelectKeystoneAccountsScreen = () => {
 
   const onNext = useCallback(() => {
     storageSelectedAccounts()
-    if (hasAccounts) {
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
-      navigation.replace('TabBarNavigator', {
-        screen: 'Home',
-        params: {
-          screen: 'KeystoneAccountAssignScreen',
-        },
-      })
-    } else {
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
-      navigation.replace('OnboardingNavigator', {
-        screen: 'KeystoneNavigator',
-        params: {
-          screen: 'KeystoneAccountAssignScreen',
-        },
-      })
-    }
-  }, [hasAccounts, navigation, selected])
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    navigation.replace('OnboardingNavigator', {
```

Comment:
If `if (hasAccounts) {` is true we should navigate to the `AccountsService`

```suggestion
    navigation.replace('AccountsService', {
            params: {
          screen: 'KeystoneAccountAssignScreen',
        },
        }
```

Reply:
resolved